ssp

```diff
@@ -8285,44 +8282,15 @@ sub check_for_newrelic {
 
 }
 
-sub check_for_zamfoo {
-    my $has_zamfoo = 0;
+sub check_for_multilevel_reseller {
+    my @ml_plugins = ( 'zamfoo', 'whmreseller', 'whmphp', 'whmamp' );
```

Comment:
I would like to see some clearer standardized array creation when it's static like this. qw() is really the preferred best practice.

EX:
```
• [root@davel:/usr/local/cpanel] (hb_integration_66)# /usr/local/cpanel/3rdparty/bin/perl -MData::Dumper -e 'my @ml_plugins = qw/ zamfoo whmreseller whmphp whmamp /; print Dumper \@ml_plugins'
$VAR1 = [
          'zamfoo',
          'whmreseller',
          'whmphp',
          'whmamp'
        ];
```
This isn't something I am going to reject - but it's best practice.